packages/web/modals/wallet-select/use-selectable-wallets.ts

```
@@ -123,13 +139,13 @@ export const useSelectableWallets = ({
                 .filter(
                   (wallet) => wallet.name === AvailableCosmosWallets.Keplr
                 )
-                .map((wallet) => ({ ...wallet, mobileDisabled: false }));
+                .map((wallet) => ({ ...wallet, mobileDisabled: false }))
             }
 
             /**
              * If user is in a normal mobile browser, show only wallet connect
              */
-            return wallet.name.endsWith("mobile") ? [...acc, wallet] : acc;
+            return wallet.name.endsWith("mobile") ? [...acc, wallet] : acc
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)